channels-rke2.yaml

```diff
@@ -2123,7 +2123,14 @@ releases:
   - version: v1.28.12+rke2r1
     minChannelServerVersion: v2.8.3-alpha1
     maxChannelServerVersion: v2.9.99
-    serverArgs: *serverArgs-v1-28-11-rke2r1
+    serverArgs: &serverArgs-v1-28-12-rke2r1
+      <<: *serverArgs-v1-28-11-rke2r1
+      ingress-controller:
+        default: ingress-nginx
+        type: string
+        options:
+          - ingress-nginx
+          - traefik
```

Comment:
Should we also allow `none` option as it's documented per https://docs.rke2.io/reference/server_config#components ?
> Ingress Controller to deploy one of, none, ingress-nginx, traefik